__init__.py

```diff
@@ -1010,6 +1018,13 @@ def _render_time(self, alarm_dt):
             self.enclosure.mouth_display_png(png, x=x, y=2, refresh=False)
             x += w
 
+    def _show_alarm_ui(self, alarm_dt, alarm_name, alarm_exp=False):
+        self.gui["alarmTime"] = nice_time(alarm_dt, speech=False,
+                                          use_ampm=True)
+        self.gui["alarmName"] = alarm_name
```

Comment:
I wonder if we should uppercase this in some fashion? Sentence / title / all uppercase.
@dschweppe do you have a preference?